README.markdown

```diff
@@ -7539,6 +7540,23 @@ This API was first introduced in the `v0.8.0` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.timer.every
```

Comment:
@agentzh Seems it's difficult to do that, because we have to pass a Lua function to C, and run the Lua function in C land.
According to LuaJIT doc:
```
The LuaJIT FFI automatically generates special callback functions whenever a Lua function is converted to a C function pointer.
Callbacks are slow!
Currently only certain C function types can be used as callback functions. Neither C vararg functions nor functions with pass-by-value aggregate argument or result types are supported.
```
http://luajit.org/ext_ffi_semantics.html#callback